soapbox/people/afraser.md

```diff
@@ -0,0 +1,79 @@
+
+We often find ourselves on projects with:
+
+- mountains of tech debt
+- poor CI/CD 
+- high-friction development process
+- an inexperienced team
+- significant over-engineering
+- inconsistent implementation of designs
+
+It is not hyperbole to say that every engineer at Rocket takes great personal pride in the _art_ of building something new, but it's hard to take pride in that art and work _around_ challenges like the ones listed above. We want to do better. We want to level up the team, pay down tech-debt, fix architectural problems at their source. We want out of meetings that are a total waste of time and we want in on the meetings that will help us build a better product.
+
+None of these things can realistically be approached without first building trust. So how do we do that?
+
+## Communicate
+
+Listen, ask questions, give updates, don't miss standup. The beginning of every project should start with a "high-touch" phase. Some (maybe most) clients mistakenly think we have everything we need before we do, make sure you reach out if you don't and request meetings with people who can answer your questions.
+
+## Deliver on promises
+
+This goes without saying, but building trust means working on what you said you're working on and delivering again and again.
+
+## Explain how things made people happy or unhappy
+
+When justifying any decision, it's rarely enough to cite "literature" or "experience" and leave it at that. People need to hear _why_. This is useful when arguing for a particular engineering or design decision, or when considering changes to things like scrum process or meeting format.
+
+## Build it, then show and tell.
+
+New ideas are much more likely to get pulled in if they're already basically done. This is particuarly useful on slow-moving teams.
+
+## Make developers' lives easier.
+
+Automate things. Improve performance. Find pain points and fix them.
+
+## Transcend your team.
+
+Get decision makers on your side and/or find a wolf pack. Communicate with and become best friends with Design, UX, Product Owners, Architecture, DevOps, etc.
+
+## Pay attention to detail.
+Don’t get sloppy (slow code, bad naming, missed edge cases, etc.). Delight developers, designers and users. That said, don't let the details derail you from delivering on your promises.
+
+- Think hard about naming. 
+- Use sensible abstractions and try to write “idiomatic” code adhering to framework patterns.
+- Provide sensible error handling for developers and users.
+- Always consider worst-case performance.
+- Always have a single source of truth for application state.
+- Write unit tests.
+- Avoid hacks and “magic numbers”. If you can’t, leave notes in comments for future travellers.
+- Employ responsive styles and sensible css transitions.
+- Optimize for the intended platform and user: Phone or desktop? Consumer or professional?
+- Avoid files with over 200 lines of code
```

Comment:
I think this needs a little more explanation, I think I know what you're after but could use a little expansion

Reply:
I just went ahead and removed it... these notes are mostly aimed at senior devs and I'm probably trying to state the obvious.